core/templates/dev/head/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -9,13 +10,29 @@
   admin-navbar .oppia-navbar-profile-admin .open > a:hover,
   admin-navbar .oppia-navbar-profile-admin .open > a:focus,
   admin-navbar .oppia-navbar-tabs-admin > li > a:hover,
-  admin-navbar .oppia-navbar-tabs-admin > li > a.active {
+  admin-navbar .oppia-navbar-tabs-admin > li > a.active,
+  admin-navbar .oppia-admin-page-dropdown-icon {
     background-color: #fff;
     color: #00376d;
   }
   admin-navbar .oppia-navbar-admin {
     top: 0;
   }
+  @media screen and (max-width: 710px) {
+    admin-navbar .oppia-admin-page-dropdown {
+      display: block;
+    }
+    admin-navbar .oppia-tabs-to-hide {
```

Comment:
Looking at this again, I think you want to semantically convey that this is only shown in <= 710px screens, so maybe change the CSS class to:

.oppia-hide-in-small-width-screens

Reply:
Changing **.oppia-tabs-to-hide** to **.oppia-hide-in-small-width-screens**. Does the remaining part look good to you?

Reply:
I have no other code comments, but I'm deferring to @nithusha21 for the full review.

Reply:
Cool, I asked so that changes can be made in a go. :+1: